driver-core/src/main/com/mongodb/client/model/expressions/Expressions.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.expressions;
+
+import org.bson.BsonBoolean;
+import org.bson.BsonInt32;
+import org.bson.BsonString;
+
+public final class Expressions {
+
+    private Expressions() {}
+
+    /**
+     * Returns an expression having the same logical boolean value as the
+     * provided boolean primitive.
+     *
+     * @param of the boolean primitive
+     * @return the boolean expression
+     */
+    public static BooleanExpression ofBoolean(final boolean of) {
```

Comment:
Could/should these be static methods on the `BooleanExpression` interface instead?  If so, a single

    static Boolean Expression of(boolean b) { ...}

would suffice.

Reply:
If we use `of` for everything, there would be conflicts on static import, so I don't think we should.

If we put these on different interfaces, the user cannot import `Expressions.*`, or scan through all of these "literals" from one file. Putting `ofBoolean` on BoolEx might make it a bit easier to find. I think perhaps we should come back to this and see how it looks when more expressions are implemented.

Reply:
Discussed on the draft PR.  Resolving